app_grim_reefer/debug.conf

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This is a Kconfig fragment which can be used to enable debug-related options
+# in the application. See the README for more details.
+
+# compiler
+CONFIG_DEBUG_OPTIMIZATIONS=y
+
+# console
+CONFIG_CONSOLE=y
+
+# UART console
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+
+# logging
+CONFIG_LOG=y
+CONFIG_APP_LOG_LEVEL_DBG=y
```

Comment:
```suggestion
CONFIG_APP_GRIM_REEFER_LOG_LEVEL_DBG=y
```

Reply:
fixed in [6cf8032](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/6cf803229d38690de83d6db332228645b0495534)